### PR TITLE
Manage media

### PR DESCRIPTION
**Pull request by JustAnOtherCoder13**
add media when save or update property work well

how could i make AddPropertyFragment lighter?

### CODE REVIEW

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        requireContext.requireActivity().sendBroadcast(mediaScanIntent);
+    }
+
+
+    public File createImageFile() throws IOException {
+        String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss", Locale.FRANCE).format(new Date());
+        String imageFileName = "JPEG_" + timeStamp + "_";
+        File storageDir = requireContext.requireActivity().getExternalFilesDir(Environment.DIRECTORY_PICTURES);
+        File image = File.createTempFile(
+                imageFileName,
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java:55]: use png

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        return image;
+    }
+
+    public void dispatchTakePictureIntent() {
+
+        Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+        File photoFile = null;
+        try {
+            photoFile = createImageFile();
+        } catch (IOException ex) {
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java:68]: toast error

**Review by fodeguirassy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java (hunk 1 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java:55]: done

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java (hunk 2 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/ManageImageHelper.java:68]: done

**Comment by JustAnOtherCoder13** [comment]
some little bug to fix, on returned media list, will certainly be ok tomorow

**Review by fodeguirassy — APPROVED** [review]

**Pull request merged**